docs/deploy-worker-udf-binaries.md

```diff
@@ -0,0 +1,101 @@
+      <td>Arbitrary Spark configuration property in <code>key=value</code> format.</br><i>e.g. spark.yarn.appMasterEnv.DOTNET_WORKER_DIR=.\worker\Microsoft.Spark.Worker-0.6.0</i></td>
+  </tr>
+  <tr>
+    <td><b>--files</b></td>
+    <td>Comma-separated list of files to be placed in the working directory of each executor.</br>
+      <li>Please note that this option is only applicable for yarn mode.</li>
+      <li>It supports specifying file names with # similar to Hadoop.</br>
+      <i>e.g. <code>myLocalSparkApp.dll#appSeen.dll</code>. Your application should use the name as <code>appSeen.dll</code> to reference <code>myLocalSparkApp.dll</code> when running on YARN.</i></li></td>
+  </tr>
+  <tr>
+    <td><b>--archives</b></td>
+    <td>Comma separated list of archives to be extracted into the working directory of each executor.</br>
+      <li>Please note that this option is only applicable for yarn mode.</li>
+      <li>It supports specifying file names with # similar to Hadoop.</br>
+      <i>e.g. <code>hdfs://&lt;path to your worker file&gt;/Microsoft.Spark.Worker.net461.win-x64-0.6.0.zip#worker</code>. This will copy and extract the zip file to <code>worker</code> folder.</i></li></td>
+  </tr>
+  <tr>
+    <td><b>application-jar</b></td>
+    <td>Path to a bundled jar including your application and all dependencies.</br>
+    <i>e.g. hdfs://&lt;path to your jar&gt;/microsoft-spark-2.4.x-0.6.0.jar</i></td>
+  </tr>
+  <tr>
+    <td><b>application-arguments</b></td>
+    <td>Arguments passed to the main method of your main class if any</td>
+  </tr>
+</table>
+
+> Note: Please specify all the `--options` before `application-jar` when launching applications with `spark-submit` otherwise they will be ignored. Please see more `spark-submit` options [here](https://spark.apache.org/docs/latest/submitting-applications.html) and running spark on YARN details [here](https://spark.apache.org/docs/latest/running-on-yarn.html).
+
+## FAQ
+#### 1. Question: When I run a spark app with UDFs, I get the following error. What should I do?
+> **Error:** [ ] [ ] [Error] [TaskRunner] [0] ProcessStream() failed with exception: System.IO.FileNotFoundException: Assembly 'mySparkApp, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' file not found: 'mySparkApp.dll'
+
+**Answer:** Please check if the `DOTNET_ASSEMBLY_SEARCH_PATHS` environment variable is set correctly. It should be the path where contains your 'mySparkApp.dll'.
+
+#### 2. Question: After I upgrade my Spark Dotnet version, and reset `DOTNET_WORKER_DIR` environment variable. Why I still get the following error?
+> **Error:** Lost task 0.0 in stage 11.0 (TID 24, localhost, executor driver): java.io.IOException: Cannot run program "Microsoft.Spark.Worker.exe": CreateProcess error=2, The system cannot find the file specified.
+
+**Answer:** Please try to restart your powershell window (or other command windows) first to take the latest environment variable values. And then start your program.
```

Comment:
```suggestion
**Answer:** Please try restarting your PowerShell window (or other command windows) first so that it can take the latest environment variable values. Then start your program.
```